pkg/util/util.go

```diff
@@ -26,6 +26,16 @@ import (
 	"golang.org/x/exp/maps"
 )
 
+// SliceContainsOnly checks that all values of a slice are a single value.
```

Comment:
Consider adding another sentence like "Returns true when the input is an empty slice."

Reply:
Hm, I would expect it to be false, updated code, test, and comment.